test/integration/config-devtool-dev/next.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   webpack: (config) => {
-    config.devtool = false
+    config.devtool = 'cheap-module-source-map'
```

Comment:
`false` is now actually fine since that would still result in `eval-source-map` being used due to pushing our fork directly to the used Webpack plugins. It's definitely a regression we can revisit but it's not worth the refactoring now. Will revisit after error DX is fixed.